docs/api/RESTAPI.md

```diff
@@ -886,6 +887,7 @@ Workflow query parameters
 |**name**  <br>*optional*|Returns only workflows with the specified name.  If specified multiple times, returns workflows with any of the specified names.  <br>**Pattern** : `"^[a-zA-Z][a-zA-Z0-9_]*$"`|string|
 |**start**  <br>*optional*|Returns only workflows with an equal or later start datetime.  Can be specified at most once. If both start and end date are specified, start date must be before or equal to end date.|string (date-time)|
 |**status**  <br>*optional*|Returns only workflows with the specified status.  If specified multiple times, returns workflows in any of the specified statuses.|enum (Submitted, Running, Aborting, Failed, Succeeded, Aborted)|
+|**submission**  <br>*optional*|Returns only workflows with an equal or later submission time. Can be specified at most once. If both submission time and start date are specified, submission time should be before or equal to start date.|string (date-time)|
```

Comment:
ToL: also @ruchim - are we planning to make it possible to search for submission times between X and Y? In that case, maybe it makes more sense to call this `submitted_after`?

Reply:
Presumably start and end would need to change too if we're adding that general capability.

Reply:
I don't know about change, I assume that the least disruptive thing would be to add the rightly named ones (eg `end_before` and `end_after`) but leave in the unqualified ones (eg `end`) as an unfortunate, deprecated option for at least a few releases so that callers get a chance to update)?

Reply:
@cjllanwarne  We definitely have use cases for supporting time ranges. I don't think its required as a part of this issue, and Miguels totally right that we'd likely need to support time range queries for start/end timestamps as well.